src/server/services/VirusScanService.ts

```diff
@@ -0,0 +1,77 @@
+import fetch from 'cross-fetch'
+import CrossFormData from 'form-data'
+
+import { injectable } from 'inversify'
+import { logger, virusTotalKey } from '../config'
+import { VirusScanServiceInterface } from './interfaces/VirusScanServiceInterface'
+
+const VIRUS_TOTAL_ENDPOINT = 'https://www.virustotal.com/api/v3'
```

Comment:
actually, i believe we're going with cloudmersive - can we delete all this code?

Reply:
Sure, let me have that done